Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apk --no-cache upgrade && \
       rm -fr /usr/share/ri
 
 RUN wget -q -O /tmp/docker.tgz \
-    https://download.docker.com/linux/static/stable/x86_64/docker-17.12.1-ce.tgz && \
+    https://download.docker.com/linux/static/stable/x86_64/docker-20.10.9.tgz && \
```

Comment:
why we missed the `-ce` (community edition) part ? not offered any more ?

Reply:
Yep, not offered anymore, here is the list: https://download.docker.com/linux/static/stable/x86_64/

I searched for a changelog explaining those naming change, but couldn't find a reason, just the dialog box here: https://docs.docker.com/engine/release-notes/18.09/